src/_igraph/graphobject.c

```diff
@@ -13895,6 +13895,46 @@ PyObject *igraphmodule_Graph_random_walk(igraphmodule_GraphObject * self,
   }
 }
 
+/**********************************************************************
+ * Other methods                                                      *
+ **********************************************************************/
+
+ /**
+ * Fluid communities
+ */
+PyObject *igraphmodule_Graph_community_fluid_communities(igraphmodule_GraphObject *self,
+                                                        PyObject *args, PyObject *kwds) {
+    static char *kwlist[] = {"no_of_communities", NULL};
+    long no_of_communities_long;
```

Comment:
Please do not use `long` in igraph code. The size of this type is unpredictable. For example, on 64-bit Windows it is only 32-bit wide.

The best thing to do when you're unsure what to use is to look at similar code in this file and borrow its approach. _Do not rely on AI assistants for this!_

In this case, you can use `Py_ssize_t` with the `n` format specifier. See https://docs.python.org/3/c-api/arg.html#numbers